### ISSUE TITLE
Order is inconsistent

### ISSUE THREAD

**Issue by getaaron** [nakiostudio/xcov#81]
On some runs I get:

```
| MTDates.framework                 | 12% |
| OMGHTTPURLRQ.framework            | 0%  |
| AFNetworking.framework            | 6%  |
```

On other runs I get:

```
| OMGHTTPURLRQ.framework            | 0%  |
| AFNetworking.framework            | 6%  |
| MTDates.framework                 | 12% |
```

It would be nice if there were a consistent (and customizable) sort order. For example, first by percentage of coverage, then by alphabetical order.

**Comment by nakiostudio** [comment]
Hey @getaaron, thanks for your feedback! 

I would go for either sorting it alphabetically or by coverage. Adding a new option for this would be overcomplicating a bit the list of arguments. What do you think?

**Comment by nakiostudio** [comment]
Change included in `1.3.0` :)
